utils/crypto/bls/public.go

```diff
@@ -70,3 +70,11 @@ func Verify(pk *PublicKey, sig *Signature, msg []byte) bool {
 func VerifyProofOfPossession(pk *PublicKey, sig *Signature, msg []byte) bool {
 	return sig.Verify(false, pk, false, msg, ciphersuiteProofOfPossession)
 }
+
+func DeserializePublicKey(pkBytes []byte) *PublicKey {
+	return new(PublicKey).Deserialize(pkBytes)
+}
+
+func SerializePublicKey(key *PublicKey) []byte {
```

Comment:
You are correct. The impetus for this change though is to move all the methods into the `bls` so that only the `bls` pkg shows these errors:

<img width="785" alt="image" src="https://github.com/ava-labs/avalanchego/assets/7675102/c1c52b06-41a5-4268-87ef-c72a3be2f15f">